### DISCUSSION TITLE
how to get spdlog to print local time

### DISCUSSION THREAD

**Discussion by AnloWho** [gabime/spdlog#2382] [Q&A]
_I have this issue that all time stamps are 1 hour off from my current computer time in our software (linux and windows alike). How to go about having spdlog outputting the correct system time.?_

let me re-phrase, the time stamp doesn't pick-up daylight saving. I found the enum to switch between utc and local, but the default local setting ignores daylight saving settings on windows 10 at least.

**Comment by tt4g** [comment]
There is no support daylight saving time because C/C++11 API does not support it.

Related #827

**Reply by tt4g** [reply]
One workaround is to create a custom formatter to calculate daylight saving time from `std::tm`.
Wiki: https://github.com/gabime/spdlog/wiki/3.-Custom-formatting#extending-spdlog-with-your-own-flags

**Reply by AnloWho** [reply]
Thanks for quick reply. Yeah I will probably have to implement something like that.
